openpype/tools/project_manager/project_manager/window.py

```diff
@@ -245,7 +245,8 @@ def _on_add_asset(self):
         self.hierarchy_view.add_asset()
 
     def _on_add_task(self):
-        self.hierarchy_view.add_task()
+        # Colorbleed edit: force the task to directly be in edit mode
+        self.hierarchy_view._add_task_and_edit()
```

Comment:
Please change `_add_task_and_edit` to `add_task_and_edit`. Otherwise approved.

Reply:
Should be resolved.